components/TextFields/examples/TextFieldOutlinedExample.m

```diff
@@ -128,8 +128,8 @@ - (void)viewDidLoad {
 
   self.messageController =
       [[MDCTextInputControllerOutlinedTextArea alloc] initWithTextInput:textFieldMessage];
-  textFieldMessage.text = @"This is where you could put a multi-line message like an email. It can"
-      "even handle new lines./n";
+  textFieldMessage.text = @"This is where you could put a multi-line message like an email.\nIt can"
```

Comment:
I think this conflicts with what I just merged.

Reply:
Yep. Resolved.